drivers/hap_report/app/views/hap_report/warehouse_reports/hap_reports/_filter.haml

```diff
@@ -0,0 +1,14 @@
+= simple_form_for @report, url: hap_report_warehouse_reports_hap_reports_path do |f|
+  - content_for :filters_col_full do
+    .row
+      .col-sm-3
+        = f.input :start_date, as: :date_picker
+      .col-sm-3
+        = f.input :end_date, as: :date_picker
+      .col-sm-6
+        = f.input :project_ids, as: :select_two, collection: HapReport::Report.hap_projects(current_user), input_html: { multiple: true }, label: 'Projects'
```

Comment:
This should really use FilterBase if possible.  It's overkill, but it will make maintenance of it much easier long-term.